tests/sles4sap/netweaver_ascs_install.pm

```diff
@@ -48,7 +48,13 @@ sub run {
     # SAP profile and solution are configured in the system
     assert_script_run "tuned-adm profile sap-netweaver";
     assert_script_run "saptune solution apply NETWEAVER";
-    assert_script_run q/kill -1 $(ps aux|grep systemd-logind|awk '{print $2}'|head -1)/;
+    assert_script_run "systemctl restart systemd-logind.service";
+
+    # If running in DESKTOP=gnome, systemd-logind restart may cause the graphical console to
+    # reset and appear in SUD, so need to select 'root-console' again
+    wait_still_screen;
```

Comment:
And what do you need the `wait_still_screen` for when the next `select_console` call looks for the `root-console` needle?

Reply:
From what I could see, the call to `select_console 'root-console'` was executed while the consequences of the `systemctl restart systemd-logind` were still being performed in the background. Basically saw a black screen right after `systemctl restart systemd-logind` followed by the assert of `root-console`, and then a display manager login screen. After that, the test failed.

Figured the screen was still changing so I went with `wait_still_screen`, but thinking more about it, perhaps a call to `check_screen` to try to determine what's in the screen could be better. Any recommendations?

Reply:
No, that's fine. Very good explanation. I just have seen many `wait_still_screen` "just to be safe" which wastes a lot of time. In this case I don't know a better way.